### ISSUE TITLE
Forgotten Touch dropping from Om'yovra in Al'Taieu

### ISSUE THREAD

**Issue by CaptSparklez** [HomepointXI/Issues#177]
**Date & Time**: 22.08.2018

**Client Version (use `/ver` in game):** 30180801_2

**Character Name:** Hakubai

**Discord Name (if different from in game):** Haku

**Nation:** San D'Oria

**Job(level)/Sub Job(level):** THF99/DNC49

**NPC or Monster or item Name:** Vanguard Neckchopper

**Zone name:** Dynamis San D'Oria

**Coordinates (use `!where`):** -

**Item tool link (for items issues only):** 3495

**Multi-boxing? (multiple clients on same connection):** No

**Steps to reproduce:** Killed Neckchooper in NW corner of Dyna Sandy, dropped Forgotten Touch, which is only supposed to drop in Dreamlands Dyna. 

**Any additional information:** see screenshot 

![hakubai_2018 08 22_215516](https://user-images.githubusercontent.com/18505482/44538497-b9465400-a701-11e8-9601-02de6b772b75.png)

**Comment by spectre99** [comment]
The website shows 11 of the Neckchopper's in Sandy Dyna have 1% chance of dropping it, not sure if it's intentional or not since as you said the touches are only supposed to drop in dreamland dyna areas.

**Comment by EpicTaru** [comment]
This was way before my time on the server though so we'll see if Heep and/or Immo remembers why they were set to drop there.

**Comment by spectre99** [comment]
Also checked the other Forgotten items and they're all restricted to dreamland, for some reason Touch is the only one that has a 1% drop in dyna sandy.

**Comment by CaptSparklez** [comment]
Yeah I asked Aut beforehand just to make sure, it's not supposed to drop anywhere but Dreamlands

**Comment by EpicTaru** [comment]
It could be a mistake, who knows, lol. Heep/Immo will let us all know soon!

**Comment by neoffxi** [comment]
In retail, Forgotten items can drop from cities as well.

Forgotten Touch - Sandy, Valkurm
Forgotten Step - WIndy, Buburimu
Forgotten Journey - Bastok, Qufim
Forgotten Thought - Beaucedine, Tavnazia
Forgotten Hope - Xarcabard, Tavnazia

**Comment by Jessland86** [comment]
DSP is currently recoding all of Dynamis and all zones will be brought up to Neo-Dynamis retail specs when it is finished.

**Comment by EpicTaru** [comment]
This was resolved in the maintenance prior to most recent but I noticed that the Item Tool is now showing this item as dropping from Om'yovra in Al'Taieu......maybe this is on purpose???? I doubt that **_anyone_** would _**forget**_ getting a **_touch_** by one of these mobs...../shudder.......

**Comment by Heepster03** [comment]
Fixed. Next update. For reals this time.

**Comment by EpicTaru** [comment]
This issue was resolved in yesterday's maintenance. Please open a new ticket should this issue re-occur.

**Comment by EpicTaru** [comment]
This issue is re-occurring, not sure why at this point in time.

**Comment by Heepster03** [comment]
Dragontamers in Dyna-BG was linked to Dyna-Valk. Issue has been resolved.
Om'Yovra was linked to the Dragontamers in Dyna-BG as well. Issue has been resolved.
Will be in next update!
